src/components/UpcomingShiftCard/UpcomingShiftCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './UpcomingShiftCard.css';
+import Card from 'react-bootstrap/Card';
```

Comment:
Change the import statement to this
```suggestion
import { Card } from 'react-bootstrap;
import './UpcomingShiftCard.css';
```